core-java/versions/src/main/java/com/reflectoring/io/java9/TryWithResources.java

```diff
@@ -0,0 +1,37 @@
+package com.reflectoring.io.java9;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.StringReader;
+
+public class TryWithResources {
```

Comment:
Please have IntelliJ format all your code examples once more and copy them into the text again (ideally with the class name around them, so that they can find them in the code examples repository easier.